pandas/tests/arithmetic/test_datetime64.py

```diff
@@ -2161,7 +2161,9 @@ def test_dti_isub_tdi(self, tz_naive_fixture):
         [
             datetime(2011, 1, 1),
             DatetimeIndex(["2011-01-01", "2011-01-02"]),
-            DatetimeIndex(["2011-01-01", "2011-01-02"]).tz_localize("US/Eastern"),
+            DatetimeIndex(["2011-01-01", "2011-01-02"]).__getattribute__("tz_localize")(
```

Comment:
umm, I would rather you create a temporary variable here than doing this

Reply:
Do you mean just straight inside the class?